rasa/core/actions/forms.py

```diff
@@ -498,13 +496,16 @@ async def _validate_if_required(
             - the form is called after `action_listen`
             - form validation was not cancelled
         """
-        if tracker.latest_action_name == "action_listen" and tracker.active_loop.get(
-            "validate", True
-        ):
-            logger.debug(f"Validating user input '{tracker.latest_message}'")
+        # no active_loop means that it is called during activation
+        need_validation = not tracker.active_loop or (
+            tracker.latest_action_name == "action_listen"
```

Comment:
let's use the constant
```suggestion
            tracker.latest_action_name == ACTION_LISTEN_NAME
```